app/models/file_set.rb

```diff
@@ -3,4 +3,9 @@ class FileSet < ActiveFedora::Base
   include ::Hyrax::FileSetBehavior
   # GeoWorks behavior to FileSet.
   include ::GeoWorks::GeoFileSetBehavior
+
+  # override the default Hyrax create_derivatives in order to call the GeoWorks create_derivatives method
+  def create_derivatives(filename)
```

Comment:
Is it necessary to override this?  I thought that this line in the initializer was supposed to take care of it:
https://github.com/curationexperts/iris/blob/master/config/initializers/hyrax.rb#L217

Reply:
I hadn't seen that line in it. I'll revise and re-submit.